Server/src/models/entities/Subcategory.ts

```diff
@@ -20,4 +20,10 @@ export class Subcategory {
 
     @UpdateDateColumn()
     updated: Date
-}
\ No newline at end of file
+}
+
+export const selectAllSubcategoriesQuery = (manager: EntityManager) =>

```

Comment:
We are using EntityManager now? How come we have methods inside the entity?

Reply:
We have always used EntityManager for our queries. That said, looking at documentation again, it might not be needed. While this would be a simple change, it should be done for all queries at once, not these ones exclusively, and so I'd prefer to do it in a different PR for consistency's sake.